<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1789</width>
    <height>908</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SVG Generator</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="2">
    <widget class="QSpinBox" name="spinBox_width">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="value">
      <number>3840</number>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Number of &lt;/p&gt;&lt;p&gt;elements&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" rowspan="10">
    <widget class="QGraphicsView" name="graphicsView"/>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Width</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QSpinBox" name="spinBox_maxstroke">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Height</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1" colspan="2">
    <widget class="QPushButton" name="pushButton_save">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QSpinBox" name="spinBox_height">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="value">
      <number>2160</number>
     </property>
    </widget>
   </item>
   <item row="10" column="1" colspan="2">
    <widget class="QPushButton" name="pushButton_generate">
     <property name="text">
      <string>Generate</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Max stroke</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QSpinBox" name="spinBox_elements">
     <property name="value">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Elements</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="radioButton_random">
        <property name="text">
         <string>Randomized</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_rectangle">
        <property name="text">
         <string>Rectangle</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_circle">
        <property name="text">
         <string>Circle</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_ellipse">
        <property name="text">
         <string>Ellipse</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_line">
        <property name="text">
         <string>Line</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_polygon">
        <property name="text">
         <string>Polygon</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_polyline">
        <property name="text">
         <string>Polyline</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_curve">
        <property name="text">
         <string>Curve</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_gradient">
        <property name="text">
         <string>Gradient</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Min Stroke</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QSpinBox" name="spinBox_minstroke">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>spinBox_width</tabstop>
  <tabstop>spinBox_height</tabstop>
  <tabstop>spinBox_elements</tabstop>
  <tabstop>spinBox_minstroke</tabstop>
  <tabstop>spinBox_maxstroke</tabstop>
  <tabstop>radioButton_random</tabstop>
  <tabstop>radioButton_rectangle</tabstop>
  <tabstop>radioButton_circle</tabstop>
  <tabstop>radioButton_ellipse</tabstop>
  <tabstop>radioButton_line</tabstop>
  <tabstop>radioButton_polygon</tabstop>
  <tabstop>radioButton_polyline</tabstop>
  <tabstop>radioButton_curve</tabstop>
  <tabstop>radioButton_gradient</tabstop>
  <tabstop>pushButton_generate</tabstop>
  <tabstop>pushButton_save</tabstop>
  <tabstop>graphicsView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
